gateway/src/main/kotlin/ua/marchenko/gateway/common/nats/NatsConnectionConfiguration.kt

```diff
@@ -2,11 +2,12 @@ package ua.marchenko.gateway.common.nats
 
 import io.nats.client.Connection
 import io.nats.client.Nats
+import org.springframework.beans.factory.annotation.Value
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 
 @Configuration
-class NatsConnectionConfiguration {
+class NatsConnectionConfiguration(@Value("\${nats.uri}") val natsUri: String) {
```

Comment:
```suggestion
class NatsConnectionConfiguration(@Value("\${nats.uri}") private val natsUri: String) {
```
*And same for domain-service nats config